electrum/util.py

```diff
@@ -677,6 +678,15 @@ def format_satoshis(
     if len(fract_part) < num_zeros:
         fract_part += "0" * (num_zeros - len(fract_part))
     result = integer_part + DECIMAL_POINT + fract_part
+    # add whitespaces as thousands' separator for better readability of numbers
+    if add_thousands_sep:
+        integer_part = "{:,}".format(int(integer_part)).replace(',', ' ')
```

Comment:
Note: with `is_diff=True`, this line is swallowing the plus sign, as e.g. `int("+7")==7`,
and also the minus sign for values less than 1 BTC (100 million sat):
`int("-0")==0` but `int("-5")==-5`

Reply:
The main point of `is_diff` is to show the `+/-` sign in the Amount column right?

Reply:
yes, IIRC that is the main use case